src/battery_manager_laptop.cpp

```diff
@@ -0,0 +1,77 @@
+#include <ros/ros.h>
+#include <sensor_msgs/BatteryState.h>
+#include <std_msgs/String.h>
+#include <string>
+#include <sstream>
+
+
+std::string old_message = "";
+ros::Publisher speech_pub;
+
+void batteryCallback(const sensor_msgs::BatteryState::ConstPtr& msg)
+{
+    int percentage;
+    int middle = 50;
+    int low = 20;
+    int empty = 10;
+    std::string message = "";
+    std::string location = "";
+    std::ostringstream string_message;
+
+
+    if (msg->location == "hero2")
```

Comment:
We can also make the location to listen for a parameter to make it generic for hero1 and amigoX etc as well.